Cargo.toml

```diff
@@ -36,15 +32,18 @@ android_log = { version = "0.1", optional = true }
 url = "1.7.2"
 backtrace = "0.3"
 
+[dependencies.jsonrpc]
+git = "https://github.com/jb55/rust-jsonrpc"
+rev = "5e4dd4817641aa2f5b9981b4ea7f3bca52e0b3d9"
+
+[dependencies.bitcoincore-rpc]
+git = "https://github.com/jb55/rust-bitcoincore-rpc"
+rev = "f9eea91f82b3812d0c82810f74edba4259ec6939"
+
 [dependencies.elements]
 version = "0.7.1"
 optional = true
 
-[dependencies.liquid-rpc]
-git = "https://github.com/stevenroose/rust-liquid-rpc.git"
-branch = "master"
-optional = true
```

Comment:
Are you dropping Liquid support?

Reply:
ah no it there were many elements serialization issues when I upgraded rust-bitcoin and I didn't get around to fixing it. I have this on my TODO